rrat/patients/templates/patients/patients_list.html

```diff
@@ -1,22 +1,38 @@
 {% extends "layout.html" %}
 
 {% block title %}
-  Dashboard
+  Patient Dashboard
 {% endblock title %}
 
 {% block header %}
-  Patient List
+  <h1 class="text-2xl font-bold">Patient List</h1>
```

Comment:
can remove the h1 tag and juts have patient list. If you want to update  the styling you can find the h1 tag in the layout